Update account_multiple.md (#4579)
galaxyproject/training-material: faqs/galaxy/account_multiple.md

<br>
Need more disk space?

>   - Review your **User -> Preferences -> Storage Dashboard** to find and manage all of your data.

Reviewer: ```suggestion
  - Review your [**User -> Preferences -> Storage Dashboard**](https://my.galaxy.training/?path=/storage) to find and manage all of your data.
```

>   - See [this tip]({% link faqs/galaxy/account_reduce_quota_usage.md %}) for more ways to free up space in your account

Reviewer: ```suggestion
  - Read about [more ways to free up space in your account]({% link faqs/galaxy/account_reduce_quota_usage.md %})
```
